code/game/machinery/cloning.dm

```diff
@@ -44,6 +45,7 @@
 	. = ..()
 
 	countdown = new(src)
+	wires = new /datum/wires/clonepod(src)

```

Comment:
This is insufficient for hitting the opened clonepod with an assembly to open the wires menu